pylintrc

```diff
@@ -42,6 +42,7 @@ disable=R,
         misplaced-comparison-constant,
         unsubscriptable-object,
         no-member,
+        consider-using-dict-items,
```

Comment:
Same here. I think I made this change on a separate PR. Perhaps performing the rebase @vinamrabenara suggested will fix the PR.

Reply:
This happened after i manually squashed some commits. This PR needs to be rebased.

Reply:
Fixed